mystery_word.py

```diff
@@ -0,0 +1,125 @@
+from random import randint
+
+
+def print_greeting():
+    print("------------------------------------------------------------")
+    print("Welcome to Mystery Word - aka Hangman with three levels")
+    print("Easy - (4-6 chars). Normal - (6-10 chars). Hard: >10 chars")
+    print()
+    print("You will have 8 opportunities to enter a character")
+    print("------------------------------------------------------------")
+
+
+def validate_level(level):
+    if level not in ["E", "N", "H"]:
+        print("INVALID CHOICE! Valid Options are: E,e,N,n,H,h")
+        return False
+    else:
+        return True
+
+
+def get_level():
+    while True:
+        print("Enter the level desired")
+        level_read = input("E/e for easy, N/n for normal, H/h for hard  ")
+        level_read = level_read.upper()
+        pass_level = validate_level(level_read)
+        if pass_level is True:
+            return level_read
+
+
+def select_random_word(level):
+    file_location = open("/usr/share/dict/words")
+    file_content = file_location.read()
+    words = file_content.split()
+    file_location.close()
+    if level == 'E':
+        min_len = 4
+        max_len = 6
+    elif level == 'N':
+        min_len = 6
+        max_len = 10
+    else:
+        min_len = 10
+        max_len = 50
+    while True:
```

Comment:
This will totally work, but you should look at using a list comprehension here. It's what you want to filter down the list of words to ones of the size you want.